src/app/core/site-navbar/site-navbar.component.scss

```diff
@@ -77,6 +77,7 @@
 			&.active-link {
 				background-color: $nav-element-active-background-color;
 				border-left: $nav-element-active-indicator-width solid $nav-element-active-color;
+				color: $nav-element-active-color;
```

Comment:
Could we add another variable to allow setting the color separate from the border color? 

Suggest adding `$nav-element-active-border-color` to set the border color.